### ISSUE TITLE
PHPStorm

### ISSUE THREAD

**Issue by Jurigag** [Avallach7/daedalus-intellij#1]
Zainstalowałem na phpstormie z pliku jar bezpośrednio bo nie znajdywało w repozytoriach jednakże za bardzo nie widać żadnego efektu, nie rozpoznaje w ogóle plików .d w sumie.

**Comment by Szmyk** [comment]
![](http://szymonzak.pl/phpstorm.png)

"Rozwiązałem" problem (prawie). Wystarczy dodać moduły do `plugin.xml`. Chyba wystarczą tylko te:

- com.intellij.modules.lang
- com.intellij.modules.platform

Czyli początek pliku plugin.xml może na przykład wyglądać tak:

```
<idea-plugin version="2">
    <id>org.avallach.daedalus.ide</id>
    <name>Daedalus support</name>
    <version>0.5</version>
    <idea-version since-build="141.0"/>
	
    <depends>com.intellij.modules.lang</depends>
    <depends>com.intellij.modules.platform</depends>
```

@Avallach7 Na marginesie powiem, że "zgłębiam" tajniki Grammar-Kit'a, więc może pojawią się Pull requesty ode mnie z ciekawymi ficzerami :)

**Comment by Jurigag** [comment]
O dzięki :P Dobra robota.

**Comment by Szmyk** [comment]
Nie wiem, czy jest sens wysyłania Pull requesta. @Avallach7 sam dopisze, potestuje i wypuści nową wersję... mam taką nadzieję.

**Comment by Jurigag** [comment]
Spoko, akurat nie problem samemu zrobić paczkę.

**Comment by Avallach7** [comment]
Wysyłaj pull requesta, jestem za leniwy ;)

**Comment by Szmyk** [comment]
OK, może coś podziałam. A jak jest zrobiony deploy do bazy pluginów JetBrains? Pobierane jest z repo, czy sam musisz to wrzucać ręcznie?

**Comment by Jurigag** [comment]
Chyba ręcznie wrzuca jara, tak mi się wydaje

**Comment by Avallach7** [comment]
Dokładnie, niestety nie ma opcji żeby to zautomatyzować (ja nie widziałem).

**Comment by Jurigag** [comment]
https://plugins.jetbrains.com/plugin/229-pluginuploader Jedynie takie coś co ułatwia robotę ale nie automatyzuje
